DEVELOPMENT.md

```diff
@@ -476,13 +474,83 @@ which contains a QA checklist. Before publishing packages for a wider audience
 someone from the team goes through the checklist and manually tests the app,
 afterwards the team can evaluate whether the release is up to our standards.
 
-**Title:** `QA: v0.0.14 macOS`\
+**Title:** `QA: v0.0.11 macOS`\
 **Body** [QA.md][qa]
 
-**Title:** `QA: v0.0.14 Linux`\
+**Title:** `QA: v0.0.11 Linux`\
 **Body:** [QA.md][qa]
 
+## Publishing a release
+
+Once a release has passed QA, it is ready to be published. This involves a
+couple of manual steps since the macOS release has to be signed and notarized
+on a developer's macOS machine.
+
+If you haven't already, please set up an Apple developer account and signing
+certificates, see [Setting up Apple notarization][an] for more details.
+
+To build, sign and notarize a macOS dmg package of the latest version run the
+following commands:
+
+```
+cd radicle-upstream
+git checkout v0.0.11
+
+CSC_NAME="Monadic GmbH (XXXXXXXXXX)" \
+APPLE_ID="XXXXXXX@monadic.xyz" \
+APPLE_ID_PASSWORD="XXXX-XXXX-XXXX-XXXX" \
+yarn dist
+```
+
+Where:
+  - `CSC_NAME` is the name of the signing certificate
+  - `APPLE_ID` is the developer's Apple developer ID
+  - `APPLE_ID_PASSWORD` is an app specific token generated from your Apple ID
+    in the developer portal
+
+**Note**: building a release might take a while, especially the notarization
+step, because it has to upload the final package to the Apple notarization
+server. Make sure you're on a stable internet connection.
+
+Once the package is built, signed and notarized you should upload it to our
+Google Cloud Platform under
+[`builds.radicle.xyz/releases/radicle-upstream/v0.1.4`][gp].
+
+You should also copy the Linux `.AppImage` and `.snap` packages to the same
+location. You can get them from the [CI build][ar] on Buildkite.
+
+After all the packages are uploaded, update the links to those binaries on the
+[radicle.xyz download][rd] and [docs.radicle.xyz/docs/getting-started][gs]
```

Comment:
Links to the source code on Github would be nice. Ideally it would just be one location where we need to change this.

Reply:
https://github.com/radicle-dev/radicle-upstream/issues/1410